scripts/release/utils.injector.js

```diff
@@ -165,6 +178,7 @@ const generateMethod = (methodIDs, dataClass, mappedSignatures) => {
     }
 
     for (const signature of data.signatures) {
+      const mappedSignature = mappedSignatures.find((item) => item.id === signature.id);
```

Comment:
move the expression to line 196 instead without assigning it to a variable?

Reply:
Fixed.